diff -p -up ncompress-4.2.4.2/compress42.c.filenamelen ncompress-4.2.4.2/compress42.c
--- ncompress-4.2.4.2/compress42.c.filenamelen	2008-03-02 18:53:34.000000000 +0100
+++ ncompress-4.2.4.2/compress42.c	2008-03-02 18:54:13.000000000 +0100
@@ -190,7 +191,7 @@
 #	define	OBUFSIZ	BUFSIZ	/* Default output buffer size							*/
 #endif
 
-#define MAXPATHLEN 1024		/* MAXPATHLEN - maximum length of a pathname we allow 	*/
+#define MAXPATHLEN PATH_MAX 		/* MAXPATHLEN - maximum length of a pathname we allow 	*/
 #define	SIZE_INNER_LOOP		256	/* Size of the inter (fast) compress loop			*/
 
 							/* Defines for third byte of header 					*/
